db/migrate/20200512170845_add_user_to_curated_cluster.rb

```diff
@@ -1,5 +1,6 @@
 class AddUserToCuratedCluster < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
   def change
-    add_reference :curated_clusters, :user, foreign_key: true
+    add_reference :curated_clusters, :user, foreign_key: true, index: {algorithm: :concurrently}
```

Comment:
also curious about the ```index: {algorithm: :concurrently}``` that was suggested

Reply:
Same issue as the ddl transaction, this message popped up when attempting to commit the migration file. After looking into this I found a helpful article that explained this process a little further:

https://thoughtbot.com/blog/how-to-create-postgres-indexes-concurrently-in